Sample/KitchenSinkUnitTests/WindCalcTests.swift

```diff
@@ -0,0 +1,89 @@
+//
+//  Shape+Extensions.swift
+//  flight-ui-ios
+//
+//  Created by Appivate 2023
+//
+
+import Foundation
+
+import XCTest
+@testable import Kitchen_Sink
+
+class WindCalcTests: XCTestCase {
+
+    var viewModel: CrosswindCalculatorViewModel!
+
+    override func setUp() {
+        super.setUp()
+        viewModel = CrosswindCalculatorViewModel()
+    }
+
+    override func tearDown() {
+        viewModel = nil
+        super.tearDown()
+    }
+
+    func testCrosswindCalculation() {
```

Comment:
Just a comment: On naming tests, this is perfectly valid, but one of the habits I think is quite good is having the given when then in the name, so this test for example could be named `test_givenValidCrosswind_whenCalculateWindSpeeds_thenValidOutput()`